### PR TITLE
allow to remove cover photo of post

### LINKED ISSUE #11962: CmsKit- Allow to remove cover photo of post
There is no way ro remove cover photo
image

### PR DESCRIPTION
**Pull request by demirmusa**
closes #11962

### CODE REVIEW

File: modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/BlogPosts/update.js [hunk 1]
```diff
@@ -225,4 +225,21 @@ $(function () {
             }
         });
     }
+
+    $buttonRemoveCoverImage.on('click', function () {
+        abp.message.confirm(
+            l('RemoveCoverImageConfirmationMessage'),
+            function (isConfirmed) {
+                if (isConfirmed) {
+                    debugger;
```
Comment (rqx110) [inline modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/BlogPosts/update.js]: delete this line

File: modules/cms-kit/src/Volo.CmsKit.Admin.HttpApi/Volo/CmsKit/Admin/Blogs/BlogPostAdminController.cs [hunk 2]
```diff
@@ -61,4 +61,12 @@ public virtual Task<BlogPostDto> UpdateAsync(Guid id, UpdateBlogPostDto input)
     {
         return BlogPostAdminAppService.UpdateAsync(id, input);
     }
+    
+    [HttpDelete]
+    [Route("removeCoverImage/{id}")]
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Admin.HttpApi/Volo/CmsKit/Admin/Blogs/BlogPostAdminController.cs:66]: most of times, PascalCase naming is not a good approach in URLs, maybe we can do it as `remove-cover-image` or just a **DELETE** request to `{post-id}/cover-image/`

Comment (enisn) [reply modules/cms-kit/src/Volo.CmsKit.Admin.HttpApi/Volo/CmsKit/Admin/Blogs/BlogPostAdminController.cs:66]: The second looks better I think. `DELETE | .../blog-posts/123/cover-image` that request explains everything well.

what do you think?

```suggestion
    [Route("{id}/cover-image")]
```

Comment (enisn) [reply modules/cms-kit/src/Volo.CmsKit.Admin.HttpApi/Volo/CmsKit/Admin/Blogs/BlogPostAdminController.cs:66]: _This change requires re-generation of client-proxies_

**Review by enisn — CHANGES_REQUESTED** [review]
Everything looks great but Route, I've suggested some alternatives for route of endpoint

**Comment by demirmusa** [comment]
This pr's branch is opened using wrong base branch. I will reopened it with dev branch

**Pull request closed without merging**